docker/BUILD

```diff
@@ -71,7 +82,7 @@ debug_docker_build(
 )
 
 debug_docker_build(
-    entrypoint = ["/usr/local/bin/pilot"],
+    entrypoint = ["/usr/local/bin/agent"],
```

Comment:
pilot-agent seems to be unique enough. istio-agent can be confused with CA node agent.